schema.json

```diff
@@ -0,0 +1,2729 @@
+                        "widget": "singleLineText"
+                    }
+                },
+                "order": [
+                    "productVariantId",
+                    "additionalPackingInstructions"
+                ]
+            }
+        },
+        "CollectionMetadata": {
+            "default": {
+                "properties": {
+                    "collectionId": {
+                        "widget": "singleLineText"
+                    },
+                    "externalCollectionTitle": {
+                        "widget": "singleLineText"
+                    }
+                },
+                "order": [
+                    "collectionId",
+                    "externalCollectionTitle"
+                ]
+            }
+        }
+    },
+    "services": {
+        "takeshapetestsite": {
+            "title": "takeshapetestsite",
+            "namespace": "Shopify",
+            "provider": "shopify",
+            "serviceType": "graphql",
+            "authenticationType": "oauth2Bearer",
+            "options": {
+                "shop": "takeshapetestsite",
+                "endpoint": ""
+            },
+            "id": "takeshapetestsite"
+        }
+    },
+    "indexedShapes": {
+        "Shopify_Product": {
+            "queries": {
+                "list": {
+                    "name": "Shopify_products",
+                    "ignoreFields": [
+                        "availablePublicationCount",
+                        "publicationCount",
+                        "publishedOnCurrentPublication",
+                        "productPublications",
+                        "publications",
+                        "resourcePublications",
+                        "resourcePublicationsV2",
+                        "unpublishedChannels",
+                        "unpublishedPublications",
+                        "metadata"
+                    ],
+                    "objectDepthLimit": 1
+                },
+                "get": {
+                    "name": "Shopify_product",
+                    "ignoreFields": [
+                        "availablePublicationCount",
+                        "publicationCount",
+                        "publishedOnCurrentPublication",
+                        "productPublications",
+                        "publications",
+                        "resourcePublications",
+                        "resourcePublicationsV2",
+                        "unpublishedChannels",
+                        "unpublishedPublications",
+                        "metadata"
+                    ],
+                    "objectDepthLimit": 1
+                }
+            },
+            "triggers": [
+                {
+                    "type": "schedule",
+                    "query": "list",
+                    "interval": 5
+                },
+                {
+                    "type": "schedule",
+                    "query": "get",
+                    "interval": 5
```

Comment:
We do not want to trigger `get` on an interval. Instead use the webhook trigger config:
```
{
  "type": "webhook",
  "query": "get",
  "service": "my-shopify-shop",
  "events": [
    "products/create",
    "products/update",
    "products/delete"
  ]
}
```